cwru_srv/srv/get_quaternion.srv

```diff
@@ -0,0 +1,7 @@
+#This service is to get a quaternion:
+
+#query is just a bool.
+bool query
+---
+#The reply is just a quaternion
+geometry_msgs/Quaternion quatQuery
```

Comment:
I thought you were just going to be using tf2 or make a different library for doing what you want to do with quaternions?  I dont know if we need a service message for it, as it may encourage its use down the road when static transforms should be set up using tf/tf2.